### ISSUE TITLE
Show date of gift card submission

### ISSUE THREAD

**Issue by smarziano** [smartchicago/kimball#183]
- [x] Show date of submission with other gift card information on person's page and /gift_cards pages
- [x] Be able to see how many sign-up gift card submissions within a timeframe or per month

**Comment by shua123** [comment]
Added created_at start and end filters to gift card search.
https://github.com/smartchicago/kimball/commit/e90eca974033979477b00d268f60ca55808e5b69

**Comment by smarziano** [comment]
The date created has been added, as well as some filtering functionality. However, there are gift cards on staging that are not showing up when searching for a date range. Here is an example:
<img width="1223" alt="screen shot 2016-09-12 at 2 54 27 pm" src="https://cloud.githubusercontent.com/assets/5550969/18450514/f816e1ec-78f8-11e6-9cd7-9fa681e93e79.png">

While I searched for gift cards equal to 07/31/2016, and there are gift cards created on this date, it's not showing up in my search results.
